src/scenes/Event/store.js

```diff
@@ -459,30 +469,11 @@ export default class EventStore {
       return;
     }
 
-    // Trying to vote over the consensus threshold
-    const optionAmount = this.selectedOption.amount;
```

Comment:
Now we support vote over threshold?

Reply:
This code is unreachable now. Since we auto adjust the `amount` field to the maximum of the threshold, it will never hit this block.